apps/judicial-system/backend/src/app/modules/case/case.service.ts

```diff
@@ -713,7 +713,8 @@ export class CaseService {
             caseFile.key &&
             ((caseFile.category &&
               caseFilesCategories.includes(caseFile.category)) ||
-              (caseFile.category === CaseFileCategory.CASE_FILE &&
+              ((caseFile.category === CaseFileCategory.CASE_FILE ||
+                caseFile.category === CaseFileCategory.INVOICE) &&
```

Comment:
As invoices always comes from LÖKE, they will all have a police case number. Therefore this logic will not capture any invoices.

The new category `INVOICE` is better included in `caseFilesCategories` above where there is no dependence on police case number existence.

Note that the only reason the category `CASE_FILE` is handled separately is that it may or may not have a police case number and consequently either be a part of a case files record (skjalaskrá) or be a standalone file listed under "other case files" in the client..